protocol/consistencychecks_test.go

```diff
@@ -149,28 +143,51 @@ func TestVerifyKeyLookupResponseWithTB(t *testing.T) {
 	cc := NewCC(d.LatestSTR(), true, pk)
 
 	// do lookup first
-	if doRequestAndVerify(d, cc, KeyLookupType, "alice") != CheckPassed {
-		t.Fatal("Unexpected verification result")
+	if e1, e2 := lookupAndVerify(d, cc, alice, key); e1 != ReqNameNotFound || e2 != CheckPassed {
+		t.Error(e1)
+		t.Error(e2)
 	}
 
 	// register
-	if doRequestAndVerify(d, cc, RegistrationType, "alice") != CheckPassed {
-		t.Fatal("Unexpected verification result")
+	if e1, e2 := registerAndVerify(d, cc, alice, key); e1 != ReqSuccess || e2 != CheckPassed {
+		t.Error(e1)
+		t.Error(e2)
 	}
-	// do lookup in the same epoch
-	if doRequestAndVerify(d, cc, KeyLookupType, "alice") != CheckPassed {
-		t.Fatal("Unexpected verification result")
+	// do lookup in the same epoch - TB TOFU
+	if e1, e2 := lookupAndVerify(d, cc, alice, nil); e1 != ReqSuccess || e2 != CheckPassed {
+		t.Error(e1)
+		t.Error(e2)
+	}
+	// re-do the request and get the key from the response
+	// the key would be extracted from the TB
```

Comment:
Maybe don't use the `lookupAndVerify` helper so that you don't to do this twice.